pages/docs/administration/getting-started/building-docker-image.mdx

```diff
@@ -0,0 +1,32 @@
+---
+priority: 10100
+---
+
+import DocsWrapper from '@/src/docs/DocsWrapper'
+
+# Building a Docker Image
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+## Prerequisite
+
+Ensure docker daemon is running.
+
+## Note
+
+Below lists options to change application configuration during dockerbuild.
+
+- default application configuration is applied from config/default.js.
+- production.js overwrites mongo configuration.
+- production.js overwrites minio configuration.
+- production.js overwrites registry configurration.
+- build.environment: openshift //must be set for OpenShift
+- openshift.[ namespace | appPublicRoute ] //must be set for OpenShift
+- smtp.enabled: true //must be set, else Bailo does not start the s2i build
+- logging.file.enabled: false //set true to debug Bailo
```

Comment:
None of these actually affect the built Docker image.  These should all be removed.

Reply:
These are all configured at runtime!

Reply:
amended - removed as per comment.